bitnami/mongodb/values.yaml

```diff
@@ -1064,6 +1064,100 @@ persistence:
     ##
     dataSource: {}
 
+## @section Backup parameters
+## This section implements a trivial logical dump cronjob of the database.
+## This only comes with the consistency guarantees of the dump program.
+## This is not a snapshot based roll forward/backward recovery backup.
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/
+backup:
```

Comment:
This is a style comment, but could you add padding to the new parameters as done for the rest of the file? This helps increase readability. E.g.:

```suggestion
## ref: https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/
##
backup:
```